mlflow/ml-package-versions.yml

```
@@ -829,6 +829,7 @@ autogen:
   autologging:
     minimum: "0.2.36"
     maximum: "0.2.40"
+    unsupported: ["0.4.*", "0.5.*"]
```

```suggestion
    unsupported: ["==0.4.*", "==0.5.*"]
```

Can we make these [version specifiers](https://packaging.python.org/en/latest/specifications/version-specifiers)? This allows us to use `packaging.specifiers.SpecifierSet` to compare versions: https://packaging.pypa.io/en/stable/specifiers.html

---

ah nice! lemme try